inputs/test-fixtures/gauge.yaml

```
@@ -0,0 +1,34 @@
+dashboards:
+  - name: "[Test] Gauge - CPU Usage"
+    description: "Test fixture for gauge chart visualization"
+    panels:
+      # ES|QL variant
+      - type: charts
+        title: "CPU Usage Gauge"
+        grid: { x: 0, y: 0, w: 24, h: 15 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)